lib/theme-config.js

```diff
@@ -355,12 +355,12 @@ ThemeConfig.prototype.getSchema = function (callback) {
 function fetchThemeSettings(path, callback) {
     var themeSettingsRegexPattern = /\Wtheme_settings\.(.+?)\W/g;
     var themeSettings = {};
-
     Fs.readFile(path, 'utf8', function (err, content) {
         var match;
 
         if (err) {
-            return callback(err);
+            // just skip the file if we can't read it
+            return callback(null, themeSettings);
```

Comment:
Thank you for the contribution, but this is not the right solution. The solution should be to change `'**/*.html'` to `'templates/**/*.html'` on line 321 https://github.com/bigcommerce/stencil-cli/pull/257/files#diff-8da2964f3c728b5ac692995aa59fae52R321

Can you try that and submit the change?